spec/blas_spec.rb

```diff
@@ -207,6 +207,28 @@
         expect(NMatrix::BLAS.nrm2(x, 1, 3)).to be_within(err).of(nrm2)
       end
 
+      it "exposes nrm2 for 0 norm" do
```

Comment:
My only critique here is I don't understand what this means: "exposes nrm2 for 0 norm." There's a 2-norm, a frobenius norm, and several others — but a 0-norm?